wgpu-native/src/command/mod.rs

```diff
@@ -26,18 +26,10 @@ use crate::{
 use crate::{ComputePassId, RenderPassId};
 
 use back::Backend;
-use hal::{
-    Device as _,
-    command::RawCommandBuffer,
-};
+use hal::{command::RawCommandBuffer, Device as _};
 use log::trace;
 
-use std::{
```

Comment:
These changes are rather undesired. Is there a way to preserve the old style of imports?

Reply:
I will check on this weekend and revert